### PR TITLE
made dockerfile pull latest lts, and made a docker service just for prod

### PR DESCRIPTION
**Pull request by Peedee2002**
looks like the issues were possibly ironed out after dependancies were updated.

### CODE REVIEW

**Review by Snuzzn — APPROVED** [review]
lgtm

File: docker-compose.yaml [hunk 1]
```diff
@@ -40,6 +40,20 @@ services:
       - backend
     networks:
       - circles-net
+  # there is probably a better way to do this, i dont know it.
```
Comment (lhjt) [inline docker-compose.yaml]: In terms of a better way, any idea what you may think? I mean if this is just for the docker-compose, this is fine as is I think.
```suggestion
```

**Review by lhjt — CHANGES_REQUESTED** [review]
image
In terms of commits, seems generally fine (other than one comment) but this also happens to pop up on load? I am assuming that this is just a service bug, and we can commit this anyway considering that this is meant to fix the project not building rather than there being issues like this in the UI.

File: docker-compose.yaml (hunk 1 above)
Comment (Peedee2002) [reply docker-compose.yaml]: i kinda just didnt want to copy-paste, but if its ok with you, then awesome. Ill remove the comment.

**Review by lhjt — APPROVED** [review]

**Comment by Peedee2002** [comment]
> image In terms of commits, seems generally fine (other than one comment) but this also happens to pop up on load? I am assuming that this is just a service bug, and we can commit this anyway considering that this is meant to fix the project not building rather than there being issues like this in the UI.

Hmm this didnt happen for me when i ran the app. We will have to investigate this a bit more then - but otherwise, we can go ahead with this PR.

**Pull request merged**